poolmgr/gp.go

```diff
@@ -61,10 +62,13 @@ type (
 		poolInstanceId         string                // small random string to uniquify pod names
 		fetcherImage           string
 		fetcherImagePullPolicy v1.PullPolicy
+		runtimeImagePullPolicy v1.PullPolicy
 		kubernetesClient       *kubernetes.Clientset
+		fissionClient          *tpr.FissionClient
 		instanceId             string // poolmgr instance id
 		labelsForPool          map[string]string
 		requestChannel         chan *choosePodRequest
+		sharedMountPath        string
```

Comment:
Good idea to move this into the `GenericPool`. But please add a comment for this field.

(Also: this struct has gotten too big. We should break it up into some logical sub-structures. Later.)

Reply:
^ Please address this comment